Aptoide/bds-sdk
# Update logic to open Aptoide when wallet no installed

**What does this PR do?**

Verifies  aptoide's version and returns the appStoreIntent accordingly

The Install Wallet intent is automatically redirected to aptoide if its version is equal or greater than 9908 (Version Code). If lower, the user is given the opportunity to open the wallet's store page in either aptoide or google play

**Database changed?**

   No

**Where should the reviewer start?**

- helpers/WalletUtils.java
- manager/WalletUtils.java
- DialogWalletInstall.java

**How should this be manually tested?**

Test both poa (notification) and billing (install dialog) redirections to wallets page with no wallet installed using aptoide versions: 
--> greather than 9908 (automatic install should be done through Aptoide when the notification/install buttin is clicked)
--> lesser than 9908 (user should be given both options when the notification/install button is clicked, aptoide and google play)
 

  Flow on how to test this or QA Tickets related to this use-case: [APPC-1250](https://aptoide.atlassian.net/browse/APPC-<1250>)

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-1250](https://aptoide.atlassian.net/browse/APPC-<1250>)

**Questions:**

  No


**Code Review Checklist**
